README.md

```diff
@@ -35,114 +35,156 @@ collections:
+This collection contains [molecule
+scenarios](https://ansible.readthedocs.io/projects/molecule/getting-started/#molecule-scenarios)
+that allow testing of the individual roles. There are scenarios that run the
+tests on both CentOS 7 and Rocky 9.
 
-To run the tests for this CentOS 7 sceneraio:
+To run the CentOS 7 tests for the roles:
 
-```
+```shell
 molecule test -s centos7_roles
 ```
 
 This command will:
 
 - install the required Ansible roles and collections
 - create a CentOS 7 container
-- `tests/molecule/resources/prepare.yml` playbook to do any required setup for the roles
-- run the `tests/molecule/resources/converge.yml` playbook, which will run the roles in this collection
-- run `tests/molecule/resources/converge.yml` a second time to check the roles are
+- `tests/molecule/resources/shared/prepare.yml` playbook to do any required
+  setup for the roles
+- run the `tests/molecule/resources/roles/converge.yml` playbook, which will run
+  the roles in this collection
+- run `tests/molecule/resources/roles/converge.yml` a second time to check the
+  roles are
   [idempotent](https://docs.ansible.com/ansible/latest/reference_appendices/glossary.html#term-Idempotency)
-- run `tests/molecule/resources/verify.yml`
+- run `tests/molecule/resources/roles/verify.yml`
 - destroy the CentOS 7 container
 
+### Test a single role in the collection
+
+Individual roles in the collection can be tested by setting the
+`MOLECULE_RUN_TAGS` environment variable:
+
+```shell
+export MOLECULE_RUN_TAGS=provision
+molecule test -s centos7_roles
+```
+
 ### Inspecting the Container
 
-If the `molecule test` command fails at any stage, the container is immediately destroyed.
-This is due to the pre-defined sequence of actions the Molecule take when the `molecule test`
-command is invoked.
+If the `molecule test` command fails at any stage, the container is immediately
+destroyed. This is due to the pre-defined sequence of actions the Molecule take
+when the `molecule test` command is invoked.
 
 If you would like to be able to access the test container, you should instead
 use the `molecule converge` command. To run this on CentOS 7:
 
-```
+```shell
 molecule converge -s centos7_roles
 ```
 
-This will install necessary Ansible roles and collections, create the test container, and run the
-molecule playbooks. If the deployment fails, the container is not destroyed.
+This will install necessary Ansible roles and collections, create the test
+container, and run the molecule playbooks. If the deployment fails, the
+container is not destroyed.
 
 #### Access the container
 
-Once the command has finished running, you can access the container using the name
-of the scenario. To access the container for the `centos7` scenario:
+Once the command has finished running, you can access the container using the
+name of the scenario. To access the container for the `centos7_roles` scenario:
```

Comment:
For the monitoring tests there are multiple containers - is it possible to specify which container you want to log into with `molecule login`? If not, should we also mention how to login to the container using `docker exec`?

Reply:
README updated accordingly.